### PR TITLE
add notes column to levels table

### PR DESCRIPTION
**Pull request by Hamms**
And a corresponding editor to levelbuilder. Both intended to be used exclusively by levelbuilders to leave notes to themselves and each other.

May also be eventually utitilzed to sort of tag levels in a queryable way.

image

[[Fixes 114820377]](https://www.pivotaltracker.com/projects/1256652/stories/114820377)

### CODE REVIEW

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -283,6 +283,7 @@
     t.string   "type",                     limit: 255
     t.string   "md5",                      limit: 255
     t.boolean  "published",                              default: false, null: false
+    t.string   "notes",                    limit: 255
```
Comment (laurelfan) [inline dashboard/db/schema.rb]: how long notes do people want to add? Should it be a text field so it can be longer than 255 char?

Comment (Hamms) [inline dashboard/db/schema.rb]: oh gosh, rookie mistake. It should indeed!

**Comment by laurelfan** [comment]
Do you think this should be persisted to the level files so that they can be checked in to source control? I don't think we want to store things solely in the levelbuilder database itself (we are operating as if it's not critical because all of the work there is checked in daily, so the db/instance doesn't have the same backups as production databases, etc.)

**Comment by Hamms** [comment]
Hmm. On the one hand, this isn't critical stuff, but is rather intended to be a replacement for the _personal_ notes that various levelbuilders have been keeping on their levels.

On the other hand, we've discussed the possibility of developing this into something we can use as an informal tagging mechanism for some analytics work, which means we may _at some point_ want this stuff in the production DB. And I suppose it'll be a lot easier to just do that now.

**Comment by laurelfan** [comment]
LGTM

**Pull request merged**